src/database/migrations/20200928155141-create-access-token.js

```diff
@@ -1,26 +0,0 @@
-module.exports = {
-  up: async (queryInterface, Sequelize) => {
```

Comment:
Why are you deleting this file?

Reply:
It is no longer used, I was created it for storing the token on login but I was wrong with tables naming conversion and we created another migration for that.